src/components/chart/_chart.scss

```diff
@@ -8,19 +8,7 @@
         @extend .ons-u-fs-r--b;
     }
 
-    &__title {
```

Comment:
Suggestion: these classes could also be removed from the macro template if they are no longer styled